### ISSUE TITLE
Path location of Private Key inside android phone 

### ISSUE THREAD

**Issue by aletest** [mycelium-com/wallet#233]
Hello guys, after much research I still can't solve an issue so I'm asking for help here.

Mycelium wallet was running ok in a Moto E android phone.

Battery went down, put it on charge. After turning the phone ON, device rebooted a couple of times.

When Android OS load, I started getting Google Play Service errors. Some apps stopped loading anymore and I'm unable to reinstall. I'm sure it was an Android issue.

Thing is when I open Mycelium wallet, it prompts the "Welcome to Mycelium" screen:
Do you want to restore a backup, or create a new wallet?

I haven't created a backup, what I want to try is to extract the Private Key from within the phone so I can recover it.

I think it should still be inside the internal phone memory somewhere.

app info:

Mycelium Bitcoin Wallet
version 2.5.9

Storage:
Total: 14.61 MB
Application: 14.41 MB
Data: 200KB

Cache Memory: 20.00 KB

Please help

Wh

**Comment by DanielWeigl** [comment]
Its in the app-data folder - it will be probably /data/data/com.mycelium.wallet/databases/walletbacking.db  (but might be in a different base folder, depending on your phone version and setup).

Its a sqlite database file - there are various programs out there to inspect them.

But if the masterseed would be there, then mycelium would also pick it up and just start normally.

Good luck, and see it as a motivation to make a backup next time.

**Comment by Giszmo** [comment]
I closed this issue as it's most likely very specific to a certain rooted phone but I marked it `question` and if anybody wants to give a more detailed howto on recovering lost keys, here would maybe be a good place to leave it.

Else I agree with Daniel that if on startup it doesn't find the keys, you probably won't find them neither cause they are gone for good. Daniel's recovery tips are more for people that broke their screen or for other reasons can't use the app itself to get to the keys.
